packages/api/src/schema/db-schema.yaml

```diff
@@ -464,6 +464,45 @@ components:
           description:
             Timestamp (in milliseconds) at which token object was created
           example: 1587667174725
+    jwt-refresh-token:
+      type: object
+      table: jwt_refresh_token
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        userId:
+          type: string
+          index: true
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+        lastSeen:
```

Comment:
What is the use of `lastSeen` field?

Reply:
Maybe stupid question, I'm not a frontend person, but when you have multiple tabs opened, then don't each need a separate JWT token? What happens if one tab refreshes the token, but we don't allow multiple refreshes?

Reply:
Both cookies and local storage (where our frontend stores these tokens) are shared between all tabs! If it weren't like that I guess you'd need to login again on every tab you have on a website.

> The main features of localStorage are: Shared between all tabs and windows from the same origin. The data does not expire. It remains after the browser restart and even OS reboot.

Reply:
So what happens is that all tabs will share the same access (jwt) and refresh (id) tokens, and when one of them gets a 401 from the API it will use the refresh token and update the access token on the local storage. Then all other tabs will get this new access token already and won't get 401s themselves.